rust/operator-binary/src/superset_controller.rs

```diff
@@ -523,6 +529,16 @@ fn build_rolegroup_config_map(
     );
 
     let mut config_file = Vec::new();
+
+    // By removing the key from `config_properties`, we avoid pasting the Python code into a Python variable as well
+    // (which would be bad)
+    // TODO: Use public constants in operator-rs to reference `EXPERIMENTAL_FILE_HEADER` and `EXPERIMENTAL_FILE_FOOTER`
+    if let Some(header) = config_properties.remove("EXPERIMENTAL_FILE_HEADER") {
```

Comment:
Please put this into a constant, ideally in operator-rs
```
// See https://github.com/stackabletech/decisions/issues/21
pub const CONFIG_OVERRIDE_FILE_HEADER_KEY: &str = "EXPERIMENTAL_FILE_HEADER";

// same for footer
```
E.g. here https://github.com/stackabletech/operator-rs/blob/37e91d6440625e174228880eccf287b9d8665654/crates/stackable-operator/src/product_config_utils.rs#L10

Reply:
Will be done with https://github.com/stackabletech/operator-rs/pull/843

Reply:
Done with https://github.com/stackabletech/superset-operator/pull/530/commits/21c328bdb0b60bf7642c0fc9f3c735eff24b9ce6